frontend/src/pages/shipments/ShipmentList.tsx

```diff
@@ -115,10 +116,18 @@ const ShipmentList: FunctionComponent = () => {
                       <td
                         {...cell.getCellProps()}
                         className={cx('p-2 first:pl-6 last:pr-6', {
+                          'font-semibold text-navy-700 hover:underline':
+                            cell.column.Header === 'Name',
                           'bg-gray-50': cell.column.isSorted,
                         })}
                       >
-                        {cell.render('Cell')}
+                        {cell.column.Header === 'Name' ? (
+                          <Link to={`/shipment/${row.original.id}`}>
```

Comment:
Suggestion (blocking): recommend adding a function to routes.ts `shipmentShow(id: number)` that generates this string

Reply:
I like it, done